realsense_ros2_camera/src/realsense_camera_node.cpp

```diff
@@ -495,6 +499,7 @@ class RealSenseCameraNode : public rclcpp::Node
 
           if (_pointcloud && is_depth_frame_arrived && is_color_frame_arrived) {
             RCUTILS_LOG_DEBUG("publishPCTopic(...)");
+            publishAlignedDepthImg(depth_frame, t);
```

Comment:
Any reasons to introduce the new local variable "depth_frame"? Is it possible to use the existing global variable "_image[DEPTH]"?